grakn-migration/json/src/main/java/ai/grakn/migration/json/JsonMigrator.java

```diff
@@ -46,28 +46,30 @@
     private final Set<Reader> readers;
 
     public static void main(String[] args) {
-        MigrationCLI.init(args, JsonMigrationOptions::new).stream()
-                .filter(Optional::isPresent)
-                .map(Optional::get)
-                .forEach(JsonMigrator::runJson);
+        try {
+            MigrationCLI.init(args, JsonMigrationOptions::new).stream()
+                    .filter(Optional::isPresent)
+                    .map(Optional::get)
+                    .forEach(JsonMigrator::runJson);
+        } catch (Throwable throwable){
+            MigrationCLI.die(throwable.getMessage());
```

Comment:
But it throws another exception correct? I think it's a bit unintuitive to do it this way especially since all it does it rewrap the exception into a runtime exception.